src/pages/workspace/card/WorkspaceCardVBAWithECardView.js

```diff
@@ -8,46 +8,60 @@ import * as Illustrations from '../../../components/Icon/Illustrations';
 import UnorderedList from '../../../components/UnorderedList';
 import * as Link from '../../../libs/actions/Link';
 import Section from '../../../components/Section';
+import Permissions from '../../../libs/Permissions';
 
 const propTypes = {
     ...withLocalizePropTypes,
 };
 
-const WorkspaceCardVBAWithECardView = props => (
-    <Section
-        title={props.translate('workspace.card.headerWithEcard')}
-        icon={Illustrations.CreditCardsBlue}
-        menuItems={[
-            {
-                title: props.translate('workspace.common.issueAndManageCards'),
-                onPress: () => Link.openOldDotLink('domain_companycards'),
-                icon: Expensicons.ExpensifyCard,
-                shouldShowRightIcon: true,
-                iconRight: Expensicons.NewWindow,
-            },
-            {
-                title: props.translate('workspace.common.reconcileCards'),
-                onPress: () => Link.openOldDotLink(encodeURI('domain_companycards?param={"section":"cardReconciliation"}')),
-                icon: Expensicons.ReceiptSearch,
-                shouldShowRightIcon: true,
-                iconRight: Expensicons.NewWindow,
-            },
-        ]}
-    >
-        <View style={[styles.mv4]}>
-            <Text>{props.translate('workspace.card.VBAWithECardCopy')}</Text>
-        </View>
+const WorkspaceCardVBAWithECardView = (props) => {
+    const menuItems = [
+        {
+            title: props.translate('workspace.common.issueAndManageCards'),
+            onPress: () => Link.openOldDotLink('domain_companycards'),
+            icon: Expensicons.ExpensifyCard,
+            shouldShowRightIcon: true,
+            iconRight: Expensicons.NewWindow,
+        },
+        {
+            title: props.translate('workspace.common.reconcileCards'),
+            onPress: () => Link.openOldDotLink(encodeURI('domain_companycards?param={"section":"cardReconciliation"}')),
+            icon: Expensicons.ReceiptSearch,
+            shouldShowRightIcon: true,
+            iconRight: Expensicons.NewWindow,
+        },
+    ];
+    if (Permissions.canUseWallet(props.betas)) {
```

Comment:
This isn't the right beta. It should be `monthlySettlements` but the beta isn't on newDot. So you have to add that in

Reply:
updated and awaiting another review